app/controllers/api/v1/accounts/portals_controller.rb

```diff
@@ -1,20 +1,22 @@
-class Api::V1::Accounts::Kbase::PortalsController < Api::V1::Accounts::BaseController
+class Api::V1::Accounts::PortalsController < Api::V1::Accounts::BaseController
   before_action :fetch_portal, except: [:index, :create]
 
   def index
-    @portals = Current.account.kbase_portals
+    @portals = Current.account.portals
   end
 
   def show; end
 
   def create
-    @portal = Current.account.kbase_portals.create!(portal_params)
+    @portal = Current.account.portals.create!(portal_params)
   end
 
   def update
     @portal.update!(portal_params)
   end
 
+  def archive; end
```

Comment:
archived should be an attribute of the model right? so we could just have this updated in the updated endpoint itself?

Reply:
This makes sense, but in this PR I was just handling the endpoint without the Kbase namespace.
So I didn't make any changes, let me do this change.

Reply:
@sojan-official I have handled this case in the last commit and added the test case for the same.